coalib/bearlib/languages/documentation/DocstyleDefinition.py

```diff
@@ -186,13 +238,38 @@ def load(cls, language: str, docstyle: str, coalang_dir=None):
         metadata = cls.Metadata(*(str(docstyle_settings.get(req_setting, ''))
                                   for req_setting in metadata_settings))
 
+        try:
+            class_padding = cls.ClassPadding(
+                *(int(padding) for padding in tuple(
```

Comment:
In your [previous comment](https://github.com/coala/coala/pull/4557#discussion_r131541556) you said that `docstyle_settings['class_padding']` is a string like "1, 0", if this is true, `tuple("1, 0")` will be `("1", ",", " ", "0")`, which `int` will choke on. Rather use `docstyle_settings['class_padding'].split(",")`

Also I'm wondering why this isn't caught by tests.